examples/volume_slice2.py

```diff
@@ -0,0 +1,70 @@
+"""
+Render slices through a volume, by creating a 3D texture, with 2D views.
+Simple and relatively fast, but no subslices.
+"""
+
+# todo: not working ATM, a Rust assertion fails in _update_texture(),
+# let's wait til the next release of wgpu-native and try again
+
+import imageio
+import visvis2 as vv
+
+from PyQt5 import QtWidgets
+from wgpu.gui.qt import WgpuCanvas
+
+
+class WgpuCanvasWithScroll(WgpuCanvas):
+    def wheelEvent(self, event):  # noqa: N802
+        degrees = event.angleDelta().y() / 8
+        scroll(degrees)
+
+
+app = QtWidgets.QApplication([])
+
+canvas = WgpuCanvasWithScroll()
+renderer = vv.renderers.WgpuRenderer(canvas)
+scene = vv.Scene()
+
+vol = imageio.volread("imageio:stent.npz")
+nslices = vol.shape[0]
+index = nslices // 2
+
+tex_size = tuple(reversed(vol.shape))
+tex = vv.Texture(vol, dim=2, size=tex_size, usage="sampled")
```

Comment:
`dim=2`?

Reply:
Yes, if I understood this right, the way to do this is to create a 2D texture with size `wxhxn`, with `n > 1`